frontend/src/charts/AgeAdjustedTableChart.tsx

```diff
@@ -128,28 +130,42 @@ export function AgeAdjustedTableChart(props: AgeAdjustedTableChartProps) {
     );
   }
 
+  const fontSize = useFontSize();
+
+  const titleStyle = {
+    font: "Inter, sans-serif",
+    fontSize: fontSize,
+    fontWeight: "bold",
+    paddingTop: 10,
+    paddingBottom: 10,
+  };
+
   return (
     <>
       {props.data.length <= 0 || props.metrics.length <= 0 ? (
         <h1>No Data provided</h1>
       ) : (
-        <TableContainer component={Paper} style={{ maxHeight: "100%" }}>
-          <Table stickyHeader {...getTableProps()}>
-            <TableHead>
-              {headerGroups.map((group, index) => (
-                <TableHeaderRow group={group} key={index} />
-              ))}
-            </TableHead>
-            <TableBody {...getTableBodyProps()}>
-              {rows.map((row: Row<any>, index) => (
-                <TableDataRow row={row} key={index} />
-              ))}
-            </TableBody>
-            <TableFooter>
-              <TableRow />
-            </TableFooter>
-          </Table>
-        </TableContainer>
+        <>
```

Comment:
this is a good idea using <figcaption> for the table header, but to be accessible I think it meeds to be wrapped in a <figure> component. So you can just swap these React fragments `<>` for `<figure>`s and i think it'll still work well and be accessible. 

should be:
```
<figure>
<figcaption>some title</figcaption>
<table>...</table>
</figure>
```

Reply:
Cool